test/units/module_utils/docker/test_common.py

```diff
@@ -462,3 +464,31 @@ def test_dict_allow_more_present(entry):
 @pytest.mark.parametrize("entry", COMPARE_GENERIC)
 def test_compare_generic(entry):
     assert compare_generic(entry['a'], entry['b'], entry['method'], entry['type']) == entry['result']
+
+
+def test_convert_duration_to_nanosecond():
+    nanoseconds = convert_duration_to_nanosecond('5s')
+    assert nanoseconds == 5000000000
+    nanoseconds = convert_duration_to_nanosecond('1m5s')
+    assert nanoseconds == 65000000000
```

Comment:
Maybe also add a test with microseconds, so that we see that there's no rounding error somewhere inbetween :) (now there isn't, but a later change might introduce something like this)

Maybe it would also make sense to test all supported units.